server/utils/firebase_algolia_server_code/functions/test/index.test.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * Unit tests for index.js
+ * Uses dependency injection to mock algolia save/delete functions
+ * Uses sinon/chai to spy on calls and validate behavior
+ * compile with `npm run test`
+ * Having problems running this on your local? 
+ * - Download a firebase service key.
+ * - *DON'T* push this key to github please.  
+ */
+
+// Initialize the firebase test mocking environment.
+const test = require('firebase-functions-test')({
+  databaseURL: 'https://step9-2020-capstone.firebaseio.com',
+  storageBucket: 'step9-2020-capstone.appspot.com',
+  projectId: 'step9-2020-capstone',
```

Comment:
Maybe some of these, like projectId, should be their own constants at the top of the file so they're easier to change

Reply:
Since these are only used in this specific context and never used outside of this snippet, I think it's ok to just use string literals.